app/(api)/api/judges/get.js

```diff
@@ -0,0 +1,27 @@
+import { NextResponse } from 'next/server';
+
+import { getDatabase } from '@utils/db/mongoClient';
+import getQueries from '@utils/request/getQueries';
+
+export async function GET(request) {
+  try {
+    const queries = getQueries(request);
+    const db = await getDatabase();
+
+    const judge = await db
+      .collection('judges')
+      .aggregate([
```

Comment:
If you don't plan on doing any joining of collections, you don't need to use aggregate. You can just use the `.find` function. This does work as well, but not rlly what the aggregation function was meant for.